integrations/docker/images/stage-2/chip-build-java/Dockerfile

```diff
@@ -1,12 +1,12 @@
-ARG VERSION=1
+ARG VERSION=2
```

Comment:
please change https://github.com/project-chip/connectedhomeip/blob/master/integrations/docker/images/base/chip-build/version and not this.

This default is actually never used (our build scripts pick up the other one in https://github.com/project-chip/connectedhomeip/blob/master/integrations/docker/build.sh#L35)

Reply:
Thank you. Modified.